grammar.js

```diff
@@ -121,11 +121,12 @@ module.exports = grammar({
       repeat($._blank),
     ),
 
-    // Codeblock: preformatted block of lines starting with ">".
+    // Codeblock: preformatted block of lines starting with ">", followed by optional language.
     codeblock: ($) => prec.right(seq(
       '>',
+      optional(alias(token.immediate(/[a-z]+/), $.language)),
```

Comment:
Question: how tight do we want to make the regex? Should we support `json5` and `c_sharp`?